tests/10-vcdm2.js

```diff
@@ -138,7 +130,8 @@ describe('Verifiable Credentials Data Model v2.0', function() {
       // around a FIXME in vc-api-test-suite-implementations/lib/requests.js
       const presentation = require('./input/presentation-ok.json');
       vc = await issue(credential);
-      vp = await prove(presentation);
+      const options = {challenge: 'test-challenge'};
+      vp = await proveVP({presentation, options});
```

Comment:
Hmm, looks like we're not testing if implementations can create presentations, which we should be doing. This is probably ok for now, but I expect there to be pushback in the group that we're not testing whether implementations can generate presentations.